Alignment/APEEstimation/test/SkimProducer/skimProducer_cfg.py

```diff
@@ -214,6 +220,47 @@
 process.load("Alignment.APEEstimation.PrivateSkim_EventContent_cff")
 process.out.outputCommands.extend(process.ApeSkimEventContent.outputCommands)
 
+#if isData1:
+#  if options.atCern:
+#    process.out.fileName = 'root://eoscms//eos/cms/store/caf/user/ajkumar/data/DoubleMu/Run2011A_May10ReReco/apeSkim.root?svcClass=cmscafuser&stageHost=castorcms'
+#  else:
+#    process.out.fileName = '/scratch/hh/current/cms/user/ajkumar/data/alcareco/data/apeSkim.root'
+#elif isData2:
+#  if options.atCern:
+#    #process.out.fileName = 'root://eoscms//eos/cms/store/caf/user/ajkumar/data/DoubleMu/Run2011A_PromptV4/apeSkim.root?svcClass=cmscafuser&stageHost=castorcms'
+#    process.out.fileName = 'apeSkim.root'
+#  else:
+#    process.out.fileName = '/scratch/hh/current/cms/user/ajkumar/data/alcareco/data/apeSkim.root'
+#if isQcd:
+#  if options.atCern:
+#    process.out.fileName = 'root://eoscms//eos/cms/store/caf/user/ajkumar/mc/Summer11/qcd/apeSkim.root?svcClass=cmscafuser&stageHost=castorcms'
+#  else:
+#    process.out.fileName = '/scratch/hh/current/cms/user/ajkumar/data/alcareco/qcd/apeSkim.root'
+#elif isWlnu:
+#  if options.atCern:
+#    process.out.fileName = 'root://eoscms//eos/cms/store/caf/user/ajkumar/mc/Summer11/wlnu/apeSkim.root?svcClass=cmscafuser&stageHost=castorcms'
+#  else:
+#    process.out.fileName = '/scratch/hh/current/cms/user/ajkumar/data/alcareco/wlnu/apeSkim.root'
+#elif isZmumu:
+#  if options.atCern:
+#    process.out.fileName = ''
+#  else:
+#    process.out.fileName = '/scratch/hh/current/cms/user/ajkumar/data/alcareco/zmumu/apeSkim.root'
+#elif isZtautau:
+#  if options.atCern:
+#    process.out.fileName = ''
+#  else:
+#    process.out.fileName = '/scratch/hh/current/cms/user/ajkumar/data/alcareco/ztautau/apeSkim.root'
+#elif isZmumu10:
+#  if options.atCern:
+#    process.out.fileName = 'root://eoscms//eos/cms/store/caf/user/ajkumar/mc/Summer11/zmumu10/apeSkim.root?svcClass=cmscafuser&stageHost=castorcms'
+#  else:
+#    process.out.fileName = ''
+#elif isZmumu20:
+#  if options.atCern:
+#    process.out.fileName = 'root://eoscms//eos/cms/store/caf/user/ajkumar/mc/Summer11/zmumu20/apeSkim.root?svcClass=cmscafuser&stageHost=castorcms'
+#  else:
+#    process.out.fileName = ''
```

Comment:
why is this all commented.
looks like it shouldn't since you control the execution of these statements with flags anway

Reply:
It seems that these lines were already removed in an earlier iteration but got back in by copying code. They are removed again as they are not really practical anyway.